pkg/clustertree/cluster-manager/controllers/pod/leaf_pod_controller.go

```diff
@@ -22,6 +24,8 @@ import (
 	"github.com/kosmos.io/kosmos/pkg/utils/podutils"
 )
 
+var gracePeriodSeconds *int64
```

Comment:
GracePeriodSeconds can be used directly as a local variable

Reply:
> GracePeriodSeconds can be used directly as a local variable

ok